### DISCUSSION TITLE
Normalize amount

### DISCUSSION THREAD

**Discussion by globophobe** [bmoscon/cryptofeed#478] [Ideas]
Bitmex BTC-USD trade amount is dollar amount. However, Coinbase BTC-USD is notional amount. It would increase the size of the message payload, but it seems useful to send volume and notional in the websocket message.

**Comment by bmoscon** [comment]
So we'd need to calculate this as appropriate and provide this additional data? Where do you envision it being provided? In the trade data?

**Comment by globophobe** [comment]
Aplogogizes, I reconsidered, this doesn't belong in core. 

I subclassed the exchange feeds to get the data the way I want, before callbacks.
